core/tests/protractor_utils/LibraryPage.js

```diff
@@ -81,6 +82,19 @@ var LibraryPage = function() {
     return waitFor.pageToFullyLoad();
   };
 
+  this.addToPlaylist = function(index) {
```

Comment:
Maybe `addToPlaylist` should accept the exploration title. Let me know what you think?

Reply:
It searches for an exploration so it will only give 1 result so we don't need the index now.

Reply:
hence, Done!